test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -260,6 +260,30 @@ public void ElementCurveReferences_ExtractsExpectedReferences()
 
         #endregion
 
+        #region Pin settings
+        private static void GetExpectedElementPinStatus(Element element, bool value)
+        {
+            bool pinStatus = element.IsPinned;
+
+            Assert.NotNull(pinStatus);
+            Assert.IsTrue(value);
```

Comment:
We should check the value is as expected, not that it's true.
```suggestion
            Assert.AreEqual(value, pinStatus);
```